java_training/src/com/lftechnology/december31/DateTest.java

```diff
@@ -0,0 +1,141 @@
+package com.lftechnology.december31;
+
+/**
+ * Write a test application named DateTest that demonstrates class Date's
+ * capabilities.
+ * 
+ * @author nerisa
+ * 
+ */
+public class DateTest {
+
+	public static void main(String[] args) {
+		DateUtility date = new DateUtility(2, 1, 2015);
+		date.displayDate();
+		date.setDay(3);
+		date.displayDate();
+		date.setMonth(2);
+		date.displayDate();
+		date.setYear(2016);
+		date.displayDate();
+		DateUtility date2 = new DateUtility(-2, 0, 23);
```

Comment:
Rather than putting direct values like -2, 0 or what so number, try to use them as constants with proper name so that the reader can know what is it in actual.

private int month = 2; or whatever needed.